test/remote/gateways/remote_forte_test.rb

```diff
@@ -0,0 +1,150 @@
+require 'test_helper'
+
+class RemoteForteTest < Test::Unit::TestCase
+  def setup
+    @gateway = ForteGateway.new(fixtures(:forte))
+
+    @amount = 100
+    @credit_card = credit_card('4000100011112224')
+    @check = check
+    @declined_card = credit_card('1111111111111111')
+    @options = {
+      billing_address: address,
+      description: 'Store Purchase'
+    }
+  end
+
+  def test_successful_purchase
+    response = @gateway.purchase(@amount, @credit_card, @options)
+    assert_success response
+    assert_equal 'TEST APPROVAL', response.message
+  end
+
+  def test_successful_purchase_with_echeck
+    response = @gateway.purchase(@amount, @check, @options)
+    assert_success response
+    assert_equal 'APPROVED', response.message
+  end
+
+  def test_successful_purchase_with_more_options
+    options = {
+      order_id: '1',
+      ip: "127.0.0.1",
+      email: "joe@example.com",
+      address: address
+    }
+
+    response = @gateway.purchase(@amount, @credit_card, options)
+    assert_success response
+    assert_equal 'TEST APPROVAL', response.message
+  end
+
+  def test_failed_purchase
+    response = @gateway.purchase(@amount, @declined_card, @options)
+    assert_failure response
+    assert_equal 'INVALID CREDIT CARD NUMBER', response.message
+  end
+
+  def test_successful_authorize_and_capture
+    auth = @gateway.authorize(@amount, @credit_card, @options)
+    assert_success auth
+
+    sleep(7)
+
+    assert capture = @gateway.capture(@amount, auth.authorization, { transaction_id: auth.params["transaction_id"] })
```

Comment:
Looks like `transaction_id` is required?  If so, perhaps the authorization's Response.authorization should be a multi-part string.